### ISSUE TITLE
Zellij skips one extra line with Page-{Up,Down} when frames are enabled

### ISSUE THREAD

**Issue by jovandeginste** [zellij-org/zellij#1002]
**Basic information**

`zellij --version`: 0.24.0
`tput lines`: 57
`tput cols`: 237
`uname -av` or `ver`(Windows): 5.13.0-23-generic

**Further information**
1. start zellij, make sure frames are enabled (add panes as you wish, it remains reproducible)
2. print numbered lines (I did: `for i in {1..200}; do echo $i; done`); make sure you have >2x as many lines printed as you have lines on your screen
3. switch to scroll mode
4. use page-up / page-down, and see how one line (number) is skipped
    - op line on my screen says `147`
    - I press page-up
    - bottom line on my screen says `145`

Bonus:
5. disable frames
6. scroll works correctly (no skipped lines)

**Comment by jaeheonji** [comment]
This issue seems similar to this #952

**Comment by jovandeginste** [comment]
> This issue seems similar to this #952

Similar, maybe, but I'm not sure it is related... I could be completely wrong, of course :wink:
